src/cil.mli

```diff
@@ -2549,6 +2549,7 @@ val unoblivType : typ -> typ
 
 val isOblivFunc : typ -> bool
 val isOblivInt : typ -> bool
+val isOblivFloat : typ -> bool (*CHANGE*)
```

Comment:
Perhaps you could document (even if only here) what this should be changed to, and why.